README.md

```diff
@@ -51,7 +60,8 @@ As a Maker
 So that I can stay constantly tapped in to the shouty box of Chitter
 I want to receive an email if I am tagged in a Peep
 ```
-
+![](images/chitter_us1.png)
+![](images/chitter_us2.png)
```

Comment:
This README is great - love to hear about your process and what you've learned, and to see the diagrams as well